[2025-05-31T09:54:30.420-0600] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-31T09:54:30.465-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: libertyPr_health_score_1.send_to_elasticsearch scheduled__2025-05-23T10:00:00+00:00 [queued]>
[2025-05-31T09:54:30.483-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: libertyPr_health_score_1.send_to_elasticsearch scheduled__2025-05-23T10:00:00+00:00 [queued]>
[2025-05-31T09:54:30.484-0600] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-05-31T09:54:30.507-0600] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): send_to_elasticsearch> on 2025-05-23 10:00:00+00:00
[2025-05-31T09:54:30.519-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1464) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-31T09:54:30.522-0600] {standard_task_runner.py:72} INFO - Started process 1466 to run task
[2025-05-31T09:54:30.524-0600] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'libertyPr_health_score_1', 'send_to_elasticsearch', 'scheduled__2025-05-23T10:00:00+00:00', '--job-id', '165', '--raw', '--subdir', 'DAGS_FOLDER/health_score_lpr.py', '--cfg-path', '/tmp/tmptsx83p3v']
[2025-05-31T09:54:30.529-0600] {standard_task_runner.py:105} INFO - Job 165: Subtask send_to_elasticsearch
[2025-05-31T09:54:30.636-0600] {task_command.py:467} INFO - Running <TaskInstance: libertyPr_health_score_1.send_to_elasticsearch scheduled__2025-05-23T10:00:00+00:00 [running]> on host 439130e6349d
[2025-05-31T09:54:30.794-0600] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='libertyPr_health_score_1' AIRFLOW_CTX_TASK_ID='send_to_elasticsearch' AIRFLOW_CTX_EXECUTION_DATE='2025-05-23T10:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-23T10:00:00+00:00'
[2025-05-31T09:54:30.797-0600] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-31T09:54:30.854-0600] {logging_mixin.py:190} INFO - ðŸ“‹ Columns in ranking file: ['FIBER_NODE', 'CMTS_ID', 'Critical', 'Good', 'Offline', 'Warning', 'Total_Modems', 'Critical_Rate', 'Warning_Rate', 'Node_Size_Amplified', 'Health_Score', 'Attention_Priority']
[2025-05-31T09:54:33.040-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/elasticsearch/_sync/client/__init__.py:403: SecurityWarning: Connecting to 'https://10.212.56.57:9200' using TLS with verify_certs=False is insecure
  _transport = transport_class(

[2025-05-31T09:54:33.565-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-31T09:54:34.135-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.580s]
[2025-05-31T09:54:34.156-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-31T09:54:34.262-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.106s]
[2025-05-31T09:54:34.286-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-31T09:54:34.391-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.105s]
[2025-05-31T09:54:34.422-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-31T09:54:34.561-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.139s]
[2025-05-31T09:54:34.601-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-31T09:54:34.711-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.110s]
[2025-05-31T09:54:34.729-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-31T09:54:34.813-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.085s]
[2025-05-31T09:54:34.838-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-31T09:54:34.957-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.120s]
[2025-05-31T09:54:34.964-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-31T09:54:34.990-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.026s]
[2025-05-31T09:54:34.992-0600] {logging_mixin.py:190} INFO - âœ… Sent 3607 documents to Elasticsearch index 'hs_lpr'
[2025-05-31T09:54:34.994-0600] {python.py:240} INFO - Done. Returned value was: None
[2025-05-31T09:54:35.018-0600] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-31T09:54:35.019-0600] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=libertyPr_health_score_1, task_id=send_to_elasticsearch, run_id=scheduled__2025-05-23T10:00:00+00:00, execution_date=20250523T100000, start_date=20250531T155430, end_date=20250531T155435
[2025-05-31T09:54:35.120-0600] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-31T09:54:35.151-0600] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-31T09:54:35.153-0600] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-05-31T21:42:54.208-0600] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-31T21:42:54.249-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: libertyPr_health_score_1.send_to_elasticsearch scheduled__2025-05-23T10:00:00+00:00 [queued]>
[2025-05-31T21:42:54.264-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: libertyPr_health_score_1.send_to_elasticsearch scheduled__2025-05-23T10:00:00+00:00 [queued]>
[2025-05-31T21:42:54.264-0600] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-05-31T21:42:54.283-0600] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): send_to_elasticsearch> on 2025-05-23 10:00:00+00:00
[2025-05-31T21:42:54.295-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=10608) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-31T21:42:54.297-0600] {standard_task_runner.py:72} INFO - Started process 10610 to run task
[2025-05-31T21:42:54.299-0600] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'libertyPr_health_score_1', 'send_to_elasticsearch', 'scheduled__2025-05-23T10:00:00+00:00', '--job-id', '338', '--raw', '--subdir', 'DAGS_FOLDER/health_score_lpr.py', '--cfg-path', '/tmp/tmp4aucketq']
[2025-05-31T21:42:54.303-0600] {standard_task_runner.py:105} INFO - Job 338: Subtask send_to_elasticsearch
[2025-05-31T21:42:54.394-0600] {task_command.py:467} INFO - Running <TaskInstance: libertyPr_health_score_1.send_to_elasticsearch scheduled__2025-05-23T10:00:00+00:00 [running]> on host 439130e6349d
[2025-05-31T21:42:54.554-0600] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='libertyPr_health_score_1' AIRFLOW_CTX_TASK_ID='send_to_elasticsearch' AIRFLOW_CTX_EXECUTION_DATE='2025-05-23T10:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-23T10:00:00+00:00'
[2025-05-31T21:42:54.556-0600] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-31T21:42:58.571-0600] {logging_mixin.py:190} INFO - ðŸ“‹ Columns in ranking file: ['FIBER_NODE', 'CMTS_ID', 'Critical', 'Good', 'Offline', 'Warning', 'Total_Modems', 'Critical_Rate', 'Warning_Rate', 'Node_Size_Amplified', 'Health_Score', 'Attention_Priority']
[2025-05-31T21:42:58.572-0600] {logging_mixin.py:190} INFO - ðŸ•“ PTIME used for this run: 2025-05-23T12:00:00
[2025-05-31T21:43:00.671-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/elasticsearch/_sync/client/__init__.py:403: SecurityWarning: Connecting to 'https://10.212.56.57:9200' using TLS with verify_certs=False is insecure
  _transport = transport_class(

[2025-05-31T21:43:01.146-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-31T21:43:01.291-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.155s]
[2025-05-31T21:43:01.339-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-31T21:43:01.400-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.062s]
[2025-05-31T21:43:01.421-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-31T21:43:01.469-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.048s]
[2025-05-31T21:43:01.488-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-31T21:43:01.529-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.042s]
[2025-05-31T21:43:01.550-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-31T21:43:01.595-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.046s]
[2025-05-31T21:43:01.615-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-31T21:43:01.655-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.040s]
[2025-05-31T21:43:01.674-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-31T21:43:01.712-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.039s]
[2025-05-31T21:43:01.719-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-31T21:43:01.739-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.020s]
[2025-05-31T21:43:01.740-0600] {logging_mixin.py:190} INFO - âœ… Sent 3608 documents to Elasticsearch index 'hs_lpr'
[2025-05-31T21:43:01.825-0600] {python.py:240} INFO - Done. Returned value was: None
[2025-05-31T21:43:01.843-0600] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-31T21:43:01.844-0600] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=libertyPr_health_score_1, task_id=send_to_elasticsearch, run_id=scheduled__2025-05-23T10:00:00+00:00, execution_date=20250523T100000, start_date=20250601T034254, end_date=20250601T034301
[2025-05-31T21:43:01.919-0600] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-31T21:43:01.933-0600] {local_task_job_runner.py:245} INFO - ::endgroup::
